main.tf

```diff
@@ -394,10 +394,9 @@ resource "local_file" "eks_config_map_aws_auth" {
 resource "null_resource" "aws_iam_authenticator" {
   provisioner "local-exec" {
     command = <<EOF
-if [ \"$(uname)\" == \"Darwin\" ]; \
+if [ \"$(uname -s)\" == \"Darwin\" ]; \
```

Comment:
don't need to backslash any of these things:
https://stackoverflow.com/questions/68957371/how-to-run-multiple-commands-using-terraform-local-exec